tests/puller_tests.py

```diff
@@ -0,0 +1,21 @@
+import unittest
+from weatherbot.puller import should_add, get_new
+
+class TestGetTemp(unittest.TestCase):
+
+    def test_should_add_true(self):
+        a = should_add({"message": "Foo", "id": 3}, 1)
```

Comment:
You can get away without this variable, or at least give it a more meaningful name.